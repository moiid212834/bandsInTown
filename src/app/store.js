import { configureStore } from '@reduxjs/toolkit';
import themeReducer from '../features/theme/ThemeSlice';
import artistReducer from '../features/artists/ArtistSlice';
import eventsReducer from '../features/events/EventsSlice';

/**
 * Configuring the store using redux toolkit and passing reducers
 */
export const store = configureStore({
  reducer: {
    theme: themeReducer,
    artists: artistReducer,
    events: eventsReducer,
  },
});
