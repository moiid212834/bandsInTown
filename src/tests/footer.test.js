import React from 'react';
import {Provider} from 'react-redux';
import {store} from '../app/store';
import {BrowserRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from '../components/footer';

it('renders correctly', () => {
    const tree = renderer
        .create(
        <Provider store={store}>
            <Router>
                <Footer></Footer>
            </Router>
        </Provider>
    )
        .toJSON();
    expect(tree).toMatchSnapshot();
});