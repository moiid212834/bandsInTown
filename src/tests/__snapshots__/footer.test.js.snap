// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiBox-root css-i9gxme"
  style={
    Object {
      "backgroundColor": "#222",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
        "margin": "10px auto 0px",
        "padding": "10px",
      }
    }
  >
    <div
      className="MuiTypography-root MuiTypography-div css-1dqc7pm-MuiTypography-root"
    >
      Â© Copyright 2022
    </div>
  </div>
</div>
`;
